Renewal/System/itemInfo_EN.lua

```diff
@@ -256386,7 +256386,7 @@ tbl = {
 	},
 	[19589] = {
 		unidentifiedDisplayName = "Unidentified Hairband",
-		unidentifiedResourceName = "�����Ӹ���",
+		unidentifiedResourceName = "�Ӹ���",
```

Comment:
Please revert this change ^^.
![image](https://user-images.githubusercontent.com/5730576/197420911-48c28046-a7b3-418d-9a27-7b61688b0777.png)